lib/jelix-www/js/authdb_admin.js

```diff
@@ -0,0 +1,6 @@
+$("document").ready( function () {
+    $("#search-login").autocomplete({
+        source: window.location.href+'/default/autocomplete',
```

Comment:
no, you must not set url your self in JS code, because this url could change, depending on how the application is configured.

You could store the url into the template, with the `{jurl}` template function, into a data attribute of the search login field for example, and retrieving its value here to give it to autocomplete()